kubernetes/charts/edge-kubernetes/templates/_helpers.tpl

```diff
@@ -27,7 +27,29 @@ certificates:
 agent:
   name: "edgeAgent"
   type: "docker"
+  {{- if .Values.edgeAgent.env }}
```

Comment:
Did this work for public preview without changes you just made?

Reply:
These template changes were copied from the preview branch, yes.